draft-srld-teas-5g-slicing.md

```diff
@@ -239,22 +245,18 @@ The term "Transport Network" is used for disambiguation with 5G network (e.g., I
 
    * 5G Network Slicing:
 
-      The objective of 5G Network Slicing is to provide a subset of
-      resources of the whole 5G infrastructure to some users/customers,
-      applications, or Public Land Mobile Networks (PLMNs) (e.g.,
-      RAN sharing). These resources are from the TN, RAN, CN
+      Is defined by the 3GPP as an appraoch where logical networks/partitions are created, with appropriate isolation, resources and optimized topology to serve a purpose or service category or customers {{TS-28.530}}. These resources are from the TN, RAN, CN
```

Comment:
On this one, I would be a a bit cautious, since I removed any ref to 3GPP (on purpose) by Just saying 5G slicing... So we avoid any interpretation